src/group_theory/perm/basic.lean

```diff
@@ -267,6 +276,47 @@ instance perm_unique {n : Type*} [unique n] : unique (equiv.perm n) :=
 
 @[simp] lemma default_perm {n : Type*} : default (equiv.perm n) = 1 := rfl
 
+variables (e : perm α) (ι : α ↪ β)
+
+open_locale classical
+
+/-- Extend a permutation along an embedding -/
+noncomputable def of_embedding (e : perm α) (ι : α ↪ β) : perm β :=
+extend_domain e (of_injective ι.1 ι.2)
```

Comment:
@pechersky, didn't we add this then remove it again? Or does it exist somewhere else with a different name?